go/adcm/status/handlers.go

```diff
@@ -295,10 +295,67 @@ func showHost(h Hub, w http.ResponseWriter, r *http.Request) {
 		ErrOut4(w, r, "HOST_NOT_FOUND", "unknown host")
 		return
 	}
-	val, _ := h.HostStorage.get(ALL, hostId)
+	val, _ := h.HostStatusStorage.get(ALL, hostId)
 	jsonOut(w, r, val)
 }
 
+func listHost(h Hub, w http.ResponseWriter, r *http.Request) {
+	allow(w, "GET, POST")
+	result, _ := h.ListHostStorage.list()
+	jsonOut(w, r, result)
+
+}
+
+func createHost(h Hub, w http.ResponseWriter, r *http.Request) {
```

Comment:
Name "createHost" can mean a lot of things, please be more specific and maybe add a docstring